common/components/componentsBySection/FindProjects/ProjectCardsContainer.jsx

```diff
@@ -52,6 +60,35 @@ class ProjectCardsContainer extends React.Component<{||}, State> {
             />
         );
   }
+
+  _handleFetchNextPage(e: object): void {
+    e.preventDefault();
+    this.setState({current_page: this.state.current_page + 1 <= this.state.project_pages 
+      ? this.state.current_page + 1 
+      : this.state.current_page }, function () {
+      ProjectSearchDispatcher.dispatch({
+        type: 'SET_PAGE',
+        page: this.state.current_page,
+      });
+    });
+  }
+
+  _renderPagination(): React$Node {
+    if (this.state.current_page === this.state.project_pages) {
+      return null; // don't render button if we've loaded the last page
+    }
+    return (
+      this.state.projects && this.state.projects.size !== 0
+      ? <div className="page_selection_footer">
+        <button className="page_button" onClick={this._handleFetchNextPage.bind(this)}>
+          More Projects... &rarr;
```

Comment:
```suggestion
          More Projects...
```
Also, we should disable the button while new results are being loaded.

Reply:
I don't know where that &rarr; came from. I must be inserting random stuff in my keyboard confusion between ubuntu and osx...  
My instance updates so fast I can't see the button until the new projects are loaded so I'm not sure what your concern is re disabling the button while loading...

Reply:
Partially to keep people from clicking multiple times and loading multiple pages, and partially to signal to users that something has happened once they've clicked the button and are waiting for the results.  Where I've deployed the changes, there is enough lag where I think we need it.

Reply:
OK. I'll work on it.

Reply:
Current commit on this branch implements waiting for projects to load before enabling the more projects button.  Try it out on your mirror (where presumably you can see some delay) and let me know how it works. I convinced myself via strategic console.logs that it does.